Add Ongoing DAG/Update enddate for backlog DAG

sachitrajbhandari opened the pull request:
- Ongoing DAG added
- Update end date for backlog DAG to cover last processed date

omad commented on dags/nci_c3_upload_s3.py:
```diff
@@ -0,0 +1,179 @@
+    INNER JOIN agdc.dataset_location dsl ON ds.id = dsl.dataset_ref 
+    WHERE dst.name='{{ params.product }}' 
+    AND (ds.added BETWEEN 
+    (to_timestamp({{ next_execution_date.timestamp() }}) at time zone 'Australia/Canberra' - interval '1 day') 
```
I think that the `- interval '1 day'` should be removed, and the date check changed to be between `{{ execution_date.timestamp() }}` and `{{ next_execution_date.timestamp() }}`.

I'm sure this works, but it feels wrong to be arbitrarily specifying a day in two different places.

omad commented on dags/nci_c3_upload_s3.py:
```diff
@@ -0,0 +1,179 @@
+
+    python3 '{{ work_dir }}/c3_to_s3_rolling.py' \
+            -f '{{ work_dir }}/{{ params.product }}.csv' \
+            -n '{{ params.nci_dir }}' \
```
If there a long versions of these arguments, I'd prefer to see them used here. No-one else is going to know what single letter arguments for a custom command are off the top of their head.

omad commented on dags/nci_c3_upload_s3.py:
```diff
@@ -0,0 +1,179 @@
+    catchup=False,
+    schedule_interval="0 7 * * *",
+    max_active_runs=4,
+    default_view="graph",
```
`graph` was a nice default view when we weren't actually running things, but `tree` (the default) is a lot more helpful now that we're running things all the time.

omad commented on dags/nci_c3_upload_s3_backlog.py:
```diff
@@ -31,8 +31,8 @@
-    ["ga_ls7e_ard_3", datetime(1999, 5, 28), datetime(2020, 7, 31)],
-    ["ga_ls8c_ard_3", datetime(2013, 3, 19), datetime(2020, 7, 31)],
+    ["ga_ls7e_ard_3", datetime(1999, 5, 28), datetime(2020, 8, 5)],
+    ["ga_ls8c_ard_3", datetime(2013, 3, 19), datetime(2020, 8, 8)],
```
Don't change it, but it feels like a hack to have copied the entire dag and organise backlog running this way.

The Airflow CLI tool includes a `backfill` command, that could probably have been used for processing the backlog instead.

omad reviewed (approved):
Hi Sachit,

This looks okay, I've added a couple of comments for changes though. But feel free to merge this PR and make changes in a new one if you wish.

Cheers

sachitrajbhandari commented:
> Hi Sachit,
> 
> This looks okay, I've added a couple of comments for changes though. But feel free to merge this PR and make changes in a new one if you wish.
> 
> Cheers

Hi Damien,
Thanks for the feedback. I've addressed your comments in this PR as they were reasonable. Review approval please!
Cheers,
Sachit.

omad reviewed (approved):
Thanks!
